vep/src/main/resources/runVEP.sh

```diff
@@ -14,6 +14,12 @@ PATH="$PATH:$VEPDIR/samtools-1.9/:$VEPDIR/ensembl-vep/htslib"
 
 # copy the part file from S3 to local
 aws s3 cp "$S3DIR/variants/$PART" .
+extension="${PART##*.}"
+if [ "$extension" = "zst" ]
+then
+  unzstd "$PART"
+  PART="${PART%.*}"
+fi
```

Comment:
Some esoteric shell script syntax as usual, I think [this](https://github.com/broadinstitute/dig-aggregator-methods/blob/1f4fe1086487a7253c68f40cb64b28c0f8301ebb/vep/src/main/scala/VepStage.scala#L64-L65) will feed in the value to `PART`.  This code will check to see if the string following the final . in PART is zst and decompress the file if it is.  At that point both the compressed and uncompressed versions of the file will be the working directory, so we remove the compressed version and then re-assign PART so that it doesn't include .zst in its name.